pkg/interface/src/App.js

```diff
@@ -79,6 +81,21 @@ class App extends React.Component {
     this.api.local.setDark(e.matches);
   }
 
+  setFavicon() {
+    if (window.ship.length < 14) {
+      const svg = sigiljs({
+        patp: window.ship,
+        renderer: stringRenderer,
+        size: 16,
+        colors: ['#fff', '#000']
```

Comment:
Consider: we know our root identity's colours from our `window.ship` in `props.contacts['/~/default']`. Might be worth polling that?